src/ui-kit/Search/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { setSearchWord } from 'redux/actions'
+
+import { func } from 'prop-types'
+import { Input } from 'antd'
+
+import { SEARCH_DEFAULT_TEXT } from '../constants'
+import './search.scss'
+
+const { Search } = Input
+
+const SearchKit = ({setSearchWord}) => {
+
+  const setCurrentYears = (value) => {
+    setSearchWord(value)
+  }
+
+  return (
+    <div>
+      <Search
+        className="ui-kit__search"
+        placeholder={SEARCH_DEFAULT_TEXT}
+        onSearch={value => setCurrentYears(value)}
+      />
+    </div>
+  )
+}
+
+
+const mapDispatchToProps = {
+  setSearchWord
+}
```

Comment:
if this is a reusable component located inside **ui-kit** you should never attach **mapDispatchToProps** to it.
You should use a callback and call **setSearchWord** from props, this gives you better flexibility and it will allow you to reuse the component in other places as well